languages/thingtalk/en/computation.genie

```diff
@@ -242,6 +242,18 @@ compute_question = {
 
 with_arg_min_max_table = {
     !turking {
+        ('best' | 'top-rated' | 'top rated') table:with_filtered_table => {
+            if (!table.schema.out['aggregateRating.ratingValue'])
+                return null;
+            return C.makeArgMinMaxTable(table, 'aggregateRating.ratingValue', 'desc', 1);
+        };
+
+        ('best' | 'top') count:constant_Number table:with_filtered_table => {
+            if (!table.schema.out['aggregateRating.ratingValue'])
+                return null;
+            return C.makeArgMinMaxTable(table, 'aggregateRating.ratingValue', 'desc', count.value);
```

Comment:
What about big numbers? Those won't have .value, they're VarRefValue with name "__const_NUMBER"

Reply:
how do i get the actual value for that?

Reply:
You don't. You do everything with Ast.Value, instead of taking the numeric value and then wrapping it again into a NumberValue

Reply:
thanks. fixed in a467169